### ISSUE TITLE
possibility to enable viewing only

### ISSUE THREAD

**Issue by atroxix** [owncloud/documents#581]
Although viewing documents in web UI is very useful, I think enabling editing through Documents app is too risky. So I'd like to enable viewing, but not editing to avoid any issues of broken documents/formatting etc.

Please add configuration option to enable viewing only / disable editing.

**Comment by witzker** [comment]
Good idea
Will this be progressed?

**Comment by jancborchardt** [comment]
@VicDeo this seems quite important functionality. Can you make it work?

**Comment by jancborchardt** [comment]
@drtomasso @tpayen maybe you also want to take a stab at solving this? :)

**Comment by cosenal** [comment]
:+1: it makes really less sense that you can't share publicly with read-only access.
One way to implement it would be to convert the document to pdf and then share the pdf.

**Comment by PVince81** [comment]
@VicDeo how complex is this ? Is it just a matter of adding a checkbox or does it need WebODF support ?

CC @MTRichards

**Comment by VicDeo** [comment]
@PVince81 How I see it's about extending core sharing capabilities.
Not a single checkbox.

**Comment by MTRichards** [comment]
While I like the idea conceptually, and competitively it would be great to have a web viewer for office docs, I can't comment on the HOW it is done, just the requirement. I agree this would be a useful and helpful feature for ownCloud.

**Comment by jancborchardt** [comment]
Ref https://github.com/owncloud/documents/issues/232 and as well https://github.com/owncloud/core/issues/7805

This is a longstanding issue also with security impacts since it’s not expected to be editable. cc @LukasReschke

**Comment by beccon4** [comment]
It would be nice to be able to share the documents in a readable form i.e. in the way we would see them in the file browser. That works for grafics - so why shouldn't it work for odf as well.

**Comment by vjanuario** [comment]
I understand the ownCloud core sharing capabilities may not allow options in sharing. Perhaps editing options could be placed in the admin section. These would be global, but would help take care of some concerns. (Allow editing, allow editing for users, allow editing for some users, ...)

**Comment by atroxix** [comment]
I agree, even the global setting would be of very much help (I actually imagined it this way originally to).

**Comment by jancborchardt** [comment]
@beccon4 @vjanuario @atroxix since ownCloud is an open source project, everyone is welcome to contribute! :) If you can’t do it yourself, maybe you know a programmer who wants to take this up?

Otherwise it would be cool to refrain from »+1« type comments as they get sent out as notifications to all people partaking in this issue, and it doesn’t help getting things done. ;)
